js/interface.js

```diff
@@ -1224,6 +1227,18 @@ var DynamicLists = (function() {
 
         // Update source
         field.data('bs.tokenfield').$input.autocomplete({source: newSource});
+        _this.removeFocusFromTokenInput();
+      });
+
+      $('input.tokenfield').on('tokenfield:createtoken', function (event) {
+        var currentTokens = $(this).tokenfield('getTokens');
+        var isExist = _.some(currentTokens, function (item) {
```

Comment:
```suggestion
        var tokenExists = _.some(currentTokens, function (item) {
```

Reply:
@tonytlwu I'm sorry, I need to change variable name?

Reply:
@maksim-an-upplabs That's correct. Please change it.

Reply:
I see you've changed it already 👍